udfs/community/README.md

```diff
@@ -586,3 +586,61 @@ returns:
 | 4	| 40 | 6.324555320336759 |
 | 5	| 50 | 12.649110640673518 |
 
+
+<br/>
+<br/>
+<br/>
+
+# StatsLib: Statistical UDFs
+
+This section details the subset of community contributed [user-defined functions](https://cloud.google.com/bigquery/docs/reference/standard-sql/user-defined-functions) 
+that extend BigQuery and enable more specialized Statistical Analysis usage patterns. 
+Each UDF detailed below will be automatically synchronized to the `fn` dataset
+within the `bqutil` project for reference in your queries.
+
+For example, if you'd like to reference the `int` function within your query,
+you can reference it like the following:
+```sql
+SELECT bqutil.fn.int(1.684)
+```
+
+## UDFs
+
+* [kruskal_wallis](#kruskal_wallisarrstructfactor-string-val-float64)
```

Comment:
linear_regression and pvalue UDFs must also be added to README.

Please add all new UDFs to the top [UDFs list](https://github.com/GoogleCloudPlatform/bigquery-utils/tree/master/udfs/community#udfs) as well, but have them link down to their descriptions in the new statslib section you created.

Reply:
Done.